options.rb

```diff
@@ -15,38 +15,9 @@ def options(app)
     number = gets.chomp.to_i
```

Comment:
- [ ] It would be nice if you use the below method everywhere to follow the DRY rule because I have seen that you have used `print` and `gets.chomp` at many places so instead of repeating the code you can create the below method somewhere in your project and use it everywhere you are taking the input from the user :+1:

```ruby
module UserInput
  def user_input(messages)
    inputs = []
    messages.each do |msg|
      print "#{msg}: "
      inputs.push(gets.chomp)
    end
    inputs
  end
end
```